docs/smart-contracts/install/migrate-pact5.md

```diff
@@ -281,6 +281,20 @@ Loaded module m, hash D45G9PizSn59QTRyUObLesCM9IEf_xrIM5OlpgQXxQg
 In the Pact 5 output, the hash for the `m` module reflects the change in the dependencies. 
 This change is required for integrity, because otherwise, modules with the same hash can behave differently.
 
+### Function names and variable identifiers
+
+In Pact 4, it was possible to use variable identifiers with the same name as built-in functions. 
+For example, in Pact 4, you can write a function like the following that uses `identity` as a variable identifier:
+
+```pact
+(register-identity-internal user-address user-guard identity country)
```

Comment:
If you're writing a function here, add `defun`. IIRC that's `(defun (register-identity-internal user-address user-guard identity country) ...)`